content/overview/workflow/process.md

```diff
@@ -0,0 +1,209 @@
+
+### correct lens distortions
+
+All lenses introduce some artifacts (distortion, chromatic aberrations, vignetting) to the image. The [_lens correction_](../../module-reference/processing-modules/lens-correction.md) module can correct many of these issues for a wide variety of lenses. The [_chromatic aberrations_](../../module-reference/processing-modules/chromatic-aberrations.md) and [_raw chromatic aberrations_](../../module-reference/processing-modules/raw-chromatic-aberrations.md) modules can also be used to handle chromatic aberrations for lenses that are not (or only partially) supported by _lens correction_. In most cases simply enabling the _lens correction_ module will auto-detect your lens and auto-apply all available corrections.
+
+If you decide to use the _lens correction_ module, it should be enabled at the start of your edit, before you adjust exposure, since vignetting correction can alter the overall brightness of your image.
+
+### reduce noise / retain detail
+
+At a pixel level there is a trade-off to be made between the retention of fine details and the reduction/removal of sensor noise. In most situations, a small amount of noise is perfectly acceptable and will not be noticeable except when you zoom in to 100%. At this scale you are not viewing the image at a realistic size -- even when represented on a large monitor or print, noise that is obvious at high zoom factors will be virtually invisible on the final image. However, some modules further along the image pipeline (especially those that increase local contrast) may end up exaggerating any noise that is present so, again, there are trade-offs to be made.
+
+The first module you can use to manage this is [_demosaic_](../../module-reference/processing-modules/demosaic.md), which controls how the single-color (R, G or B) pixels in your Raw file are converted to image pixels combining all three colors. Each demosaic algorithm has its own trade-offs between retaining fine detail and reducing noise. The default demosaic algorithm (RCD) usually provides a reasonable compromise.
+
+Demosaic algorithms can only do so much to manage noise in your image. The [_denoise (profiled)_](../../module-reference/processing-modules/denoise-profiled.md) module is individually tuned for a number of common camera sensors and can be used to reduce or remove pixel noise. As with demosaic you should alter the settings until you are happy with the balance of denoising vs fine-detail reproduction. The default settings are usually sufficient.
```

Comment:
The purpose of this paragraph is to discuss denoising modules, so mentioning demosaicing as the very first topic, and again in the middle of this paragraph, it does not flow well.

Reply:
not sure I agree. Would you like to propose an alternative wording?

Reply:
this isn't just about denoise it's about the trade-off between noise and detail and that can be affected both by the demosaic algorithm and the denoise module.